recipes/modestpy/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "0.0.7" %}
+
+package:
+  name: modestpy
+  version: {{ version }}
+
+source:
+  fn: modestpy-{{ version }}.tar.gz
+  url: https://github.com/sdu-cfei/modest-py/archive/v{{ version }}.tar.gz
+  sha256: 687d8372540aeeb901c99dc345a1441a0c5d15874aaa365e1754353f90dca075
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
If you're making changes, you might also think about using the new standard for the `script` line:

https://conda-forge.org/docs/meta.html#use-pip